### ISSUE TITLE
OCSP error is partly localized

### ISSUE THREAD

**Issue by JustOff** [MoonchildProductions/pale-moon-localization#160]
Here is the screenshot from es-MX translator: http://imgur.com/a/gAsRn

chrome://pipnss/locale/nsserrors.properties is verified to be translated

Unfortunately I can't find the way to get this error to test manually

**Comment by MikkCZ** [comment]
I think this is https://bugzilla.mozilla.org/show_bug.cgi?id=1269792.

**Comment by JustOff** [comment]
@MikkCZ yes, thanks, you are right, and I'm afraid that simple fix can't be done because of design of error handling

**Comment by lobocursor** [comment]
Sorry for not write this sooner as I lost my password.

From the screenshot the text reads
**Conexión segura fallada** (secure connection is failed)

It should read

**Conexión segura fallida** (secure connection fail)

I am not sure where to change manually this message so I left it here.

**Comment by MikkCZ** [comment]
Hi @lobocursor .

Pale Moon localization uses [CrowdIn](https://crowdin.com/project/pale-moon/es-MX), where you can suggest the string update. To be specific, this string is most likely in nsserrors.properties or netError.dtd file.

In Firefox I see this [fixed](https://transvision.mozfr.org/?recherche=Conexi%C3%B3n+segura&repo=aurora&sourcelocale=en-US&locale=es-MX&search_type=strings_entities).

**Comment by lobocursor** [comment]
Well I made the changes here already, as these are just a change of letters for fix bad grammar, in the next commit they will pass fine. ;)

**Comment by wolfbeast** [comment]
In Firefox this isn't fixed, but Firefox uses l10n to create individual installers. This is only a problem when using language packs.

I've CCed myself to the BMO bug to keep an eye on it.

**Comment by MikkCZ** [comment]
This has been fixed in Firefox (see the last comments in the bug).

**Comment by JustOff** [comment]
Pale Monn has no localization problem with the `sec_error_unknown_issue` error page mentioned in [bug #1269792](https://bugzilla.mozilla.org/show_bug.cgi?id=1269792) and all other test pages from badssl.com.

This issue is about OCSP error, unfortunately I don't know how to reproduce it intentionally.

**Comment by MikkCZ** [comment]
I think it's the same bug, just with different error page. I have specifically reproduced the steps from bug 1269792 in the latest PM (see the text for technical details below is English).
![snimek z 2017-12-23 17-51-55](https://user-images.githubusercontent.com/3941344/34321234-21115fec-e80a-11e7-8fd3-04aa53dfc7df.png)

But regardless this is a single bug or two, it's a bug in PM itself, not in the l10n packages.

**Comment by JustOff** [comment]
Here is how the [the sec_error_unknown_issuer](https://untrusted-root.badssl.com/) error looks with Russian LP installed:

![palemoon_2017-12-23_18-58-47](https://user-images.githubusercontent.com/340855/34321267-7402ba1c-e813-11e7-8e37-9d02cef0b99d.png)

All strings are translated.

**Comment by MikkCZ** [comment]
What were your STR? I just downloaded Linux x64 build from https://linux.palemoon.org/download/mainline/, added langpack https://addons.palemoon.org/language-packs/, changed about:config and restarted. All with clear profile (I do not use PM regularly).

**Comment by MikkCZ** [comment]
Btw. I think you can reproduce the original report on https://revoked.badssl.com/. It's not OCSP, but the error page looks the same, including what string are not localized.
![snimek z 2017-12-23 18-36-38](https://user-images.githubusercontent.com/3941344/34321479-3ca720e2-e810-11e7-8502-a5317ec53047.png)

**Comment by JustOff** [comment]
> What were your STR?

All the same, but using Windows.

![palemoon_2017-12-23_19-44-23](https://user-images.githubusercontent.com/340855/34321530-da55d762-e819-11e7-9e72-8968cf1535f9.png)

**Comment by wolfbeast** [comment]
The bugzilla bug states that it was resolved "in Firefox 58 or 59" but no indication what changed it or when. Perhaps someone can run a mozregression on that rage of Firefox and find out exactly which bug was responsible.

**Comment by MikkCZ** [comment]
Yes that will be a lengthy process giving the fact how langpacks work (and break Nightly).

**Comment by janekptacijarabaci** [comment]
An example of the structure of the original URL (e.g. `https://untrusted-root.badssl.com`) is as follows (for Scratchpad e.g.: `alert(document.documentURI);`):
```
about:certerror?e=nssBadCert&u=https%3A//untrusted-root.badssl.com/&c=UTF-8&f=regular&d=untrusted-root.badssl.com%20uses%20an%20invalid%20security%20certificate.%0A%0AThe%20certificate%20is%20not%20trusted%20because%20the%20issuer%20certificate%20is%20unknown.%0A%0A%28Error%20code%3A%20sec_error_unknown_issuer%29%0A
```

AFAIK - this bug can be (the event - or this string: `&d=...` - is not captured - or just sometimes):
https://github.com/MoonchildProductions/Pale-Moon/blob/27.6.2_Release/uriloader/base/nsIWebProgressListener.idl#L338
...or somewhere else.
